server/src/graphql/resolvers/queries.js

```diff
@@ -31,4 +31,9 @@ module.exports = {
       return invalidUser;
     }
   },
+
+  getAllRigs: async(parent) => {
+    const allRigs = await db.Rig.findAll({ include: [db.Well] })
+    return allRigs
```

Comment:
you can wrap this in a try catch block in case theres an error. Also you can get rid of the declaration and just return the await statement.